### DISCUSSION TITLE
blog/posts/cursor-composer

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#330] [文档评论区]
# blog/posts/cursor-composer

Cursor AI 非官方教程社区

http://localhost:5173/blog/posts/cursor-composer.html

**Comment by geektao1024** [comment]
不错 ，点赞！

**Comment by northword** [comment]
@geektao1024 烦请在 discus 中修改您的 repo id，不要使用我们的仓库作为您站点的评论存储仓库，谢谢！

https://github.com/zotero-chinese/website/blob/7b6749a004803f83df389e2c34d1c9c6fbf31a6f/src/.vitepress/theme/components/Giscus.vue#L19-L22
